securitycenter/snippets_v2/snippets_bigquery_export_v2.py

```diff
@@ -0,0 +1,189 @@
+    from google.cloud import securitycenter_v2
+
+    """
+    Retrieve an existing BigQuery export.
+    Args:
+        parent: Use any one of the following resource paths:
+                 - organizations/{organization_id}/locations/{location_id}
+                 - folders/{folder_id}/locations/{location_id}
+                 - projects/{project_id}/locations/{location_id}
+        bigquery_export_id: Unique identifier that is used to identify the export.
+    """
+
+    client = securitycenter_v2.SecurityCenterClient()
+
+    request = securitycenter_v2.GetBigQueryExportRequest()
+    request.name = f"{parent}/bigQueryExports/{bigquery_export_id}"
+
+    response = client.get_big_query_export(request)
+    print(f"Retrieved the BigQuery export: {response.name}")
+
+
+# [END securitycenter_get_bigquery_export]
+
+
+# [START securitycenter_list_bigquery_export]
+def list_bigquery_exports(parent: str):
+    from google.cloud import securitycenter_v2
+
+    """
+    List BigQuery exports in the given parent.
+    Args:
+         parent: The parent which owns the collection of BigQuery exports.
+             Use any one of the following resource paths:
+                 - organizations/{organization_id}/locations/{location_id}
+                 - folders/{folder_id}/locations/{location_id}
+                 - projects/{project_id}/locations/{location_id}
+    """
+
+    client = securitycenter_v2.SecurityCenterClient()
+
+    request = securitycenter_v2.ListBigQueryExportsRequest()
+    request.parent = parent
+
+    response = client.list_big_query_exports(request)
+
+    print("Listing BigQuery exports:")
+    for bigquery_export in response:
+        print(bigquery_export.name)
+
+
+# [END securitycenter_list_bigquery_export]
+
+
+# [START securitycenter_update_bigquery_export]
+def update_bigquery_export(parent: str, export_filter: str, bigquery_export_id: str):
+    """
+    Updates an existing BigQuery export.
+    Args:
+        parent: Use any one of the following resource paths:
+                 - organizations/{organization_id}/locations/{location_id}
+                 - folders/{folder_id}/locations/{location_id}
+                 - projects/{project_id}/locations/{location_id}
+        export_filter: Expression that defines the filter to apply across create/update events of findings.
+        bigquery_export_id: Unique identifier provided by the client.
+        For more info, see:
+        https://cloud.google.com/security-command-center/docs/how-to-analyze-findings-in-big-query#export_findings_from_to
+    """
+    from google.cloud import securitycenter_v2
+    from google.protobuf import field_mask_pb2
+
+    client = securitycenter_v2.SecurityCenterClient()
+
+    # Set the new values for export configuration.
+    bigquery_export = securitycenter_v2.BigQueryExport()
+    bigquery_export.name = f"{parent}/bigQueryExports/{bigquery_export_id}"
+    bigquery_export.filter = export_filter
```

Comment:
Include the description field as well. Refer: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/security-command-center/snippets/src/main/java/vtwo/bigquery/UpdateBigQueryExport.java

Reply:
This filter is passed by user input.

Reply:
No, I meant, update the `bigquery_export.description` field as well. And add this path in the field mask.
Please refer to: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/security-command-center/snippets/src/main/java/vtwo/bigquery/UpdateBigQueryExport.java#L68